includes/html/graphs/sensor/auth.inc.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
+use App\Models\Sensor;
+
 if (is_numeric($vars['id'])) {
-    $sensor = dbFetchRow('SELECT * FROM sensors WHERE sensor_id = ?', [$vars['id']]);
+    $sensor = Sensor::find($vars['id']);
 
-    if (is_numeric($sensor['device_id']) && ($auth || device_permitted($sensor['device_id']))) {
+    if ($auth || device_permitted($sensor->device_id)) {
         $device = device_by_id_cache($sensor['device_id']);
```

Comment:
This needs updating to `$sensor->device_id`

Reply:
I missed it twice. THANKS!